### PR TITLE
Okta 475054 checkbox select fix

### PR DESCRIPTION
**Pull request by praveendurai-okta**
## Description:

This PR is to update the latest courage package in SIW which has a bug fix for [multi field checkbox issue](https://oktainc.atlassian.net/browse/OKTA-475054) and having the focus on select. 

courage changes - https://github.com/okta/okta-ui/pull/4940 

## PR Checklist

- [X] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
Before Fix
https://user-images.githubusercontent.com/94395085/156262576-8e8bd08c-f16e-4c56-ae33-af5062c1987b.mp4

After Fix

https://user-images.githubusercontent.com/94395085/156262589-ea9e80b5-5fbf-4f46-b028-a25745567d31.mp4

### Reviewers:
@mauriciocastillosilva-okta  , @indirathirumalaimurugan-okta 

### Issue:

- [OKTA-475054](https://oktainc.atlassian.net/browse/OKTA-475054)

### CODE REVIEW

File: assets/sass/modules/_enroll.scss [hunk 1]
```diff
@@ -430,10 +430,6 @@ $enroll-content-spacing: 15px;
       }
     }
   }
-  .chzn-drop:focus-within {
-    box-shadow: 0 0 8px $input-shadow-color-focus;
-    border-color: $input-border-color-focus;
-  }
```
Comment (indirathirumalaimurugan-okta) [inline assets/sass/modules/_enroll.scss:436]: is this new change? I dont see it in the respective [okta-ui PR](https://github.com/okta/okta-ui/pull/4940) will removing this cause any regression?

File: assets/sass/okta-sign-in.scss [hunk 2]
```diff
@@ -167,4 +167,11 @@
   }
 }
 
+.chzn-container {
+  &:focus-within {
+    box-shadow: 0 0 8px $input-shadow-color-focus;
+    border-color: $input-border-color-focus;
+  }
+}
+
```
Comment (indirathirumalaimurugan-okta) [inline assets/sass/okta-sign-in.scss:176]: have same question as above

File: assets/sass/modules/_enroll.scss (hunk 1 above)
Comment (praveendurai-okta) [reply assets/sass/modules/_enroll.scss:436]: this change is specific to SIW. none of the chosen select component has focus highlighted in SIW. especially when working with screen 
readers. Without focus, it's hard to figure out where the cursor is. 


Screen Shot 2022-03-02 at 2 04 33 PM

Comment (praveendurai-okta) [reply assets/sass/modules/_enroll.scss:436]: btw its the same highlight styles we have for input text box
Screen Shot 2022-03-02 at 2 05 46 PM
.

File: assets/sass/okta-sign-in.scss (hunk 2 above)
Comment (praveendurai-okta) [reply assets/sass/okta-sign-in.scss:176]: answered above.

File: assets/sass/modules/_enroll.scss (hunk 1 above)
Comment (indirathirumalaimurugan-okta) [reply assets/sass/modules/_enroll.scss:436]: is `.chzn-drop` no longer exist on the UI and so`.chzn-drop:focus-within` is being removed?

Comment (praveendurai-okta) [reply assets/sass/modules/_enroll.scss:436]: `.chzn-drop` is for the dropdown options and will have height and width only when select is open. we need to have focus even when the select is closed, so i have moved the rules to it's parent `.chzn-container`

**Review by indirathirumalaimurugan-okta — APPROVED** [review]
lgtm

**Review by mauriciocastillosilva-okta — APPROVED** [review]

**Comment by praveendurai-okta** [comment]
Test okta-core Bacon is green. - https://bacon-go.aue1e.saasure.net/commits?artifact=monolith&sha=ca24510beadd3e6bc2549e8e90fb2412c3c35517&page=1&pageSize=6

**Pull request closed without merging**